src/locale/dim.json

```diff
@@ -590,6 +590,8 @@
   "Milestone": {
     "Daily": "Daily Challenge",
     "OneTime": "One Time Challenge",
+    "SeasonalRank": "Seasonal Rank",
+    "SeasonEnds": "Season ends: ",
```

Comment:
```suggestion
    "SeasonEnds": "Season ends: {{date}}",
```

Reply:
I don't think this is going to work with the countdown component as the date, unless there's a different way to do this?

```js
{t('Milestone.SeasonEnds', {
  date: <Countdown endTime={new Date(seasonEnd!)} compact={true} />
})}
```

Output

```
Season ends: [object Object]
```

Reply:
That is a shame.